geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/TabularResultModel.java

```diff
@@ -68,4 +70,21 @@ public void addRow(String... values) {
       table.get(columnHeaders[i]).add(values[i]);
     }
   }
+
+  public boolean setContent(List<CliFunctionResult> functionResults, boolean skipIgnore) {
```

Comment:
I don't think this is the right place for this logic - rather just pull this up to `ResultModel`